src/plone/restapi/services/content/configure.zcml

```diff
@@ -70,6 +70,45 @@
     permission="zope2.View"
     />
 
+  <plone:service
+    method="OPTIONS"
+    name="@upload"
```

Comment:
@buchi didn't we decide to use a separate "tus-upload" endpoint in addition to a regular "upload" endpoint in Bonn?